internal/command/metrics/machine_metrics.go

```diff
@@ -0,0 +1,64 @@
+package metrics
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	tailor "github.com/yitsushi/file-tailor"
+)
+
+type MachineMetrics struct {
```

Comment:
I'll move all of these under firecracker and make `MachineMetrics` an interface.